guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -22,7 +22,7 @@ endif::[]
 .Before You Begin
 
 * Review and update your firewall configuration before upgrading your {ProjectServer}.
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html-single/installing_satellite_server_from_a_disconnected_network/index#satellite-ports-and-firewalls-requirements_satellite[Ports and Firewalls Requirements] in _Installing {ProjectServer} from a Disconnected Network_.
+For more information, see {InstallingDisconnectedDocURL}Ports_and_Firewalls_Requirements_satellite[Ports and Firewalls Requirements] in _Installing {ProjectServer} in a Disconnected Network Environment_.
```

Comment:
This hardcoded context is really weird but IMHO correct. I assume this is due to disconnected being officially unsupported for Foreman/Katello.

```
➜  foreman-documentation git:(master) git grep -n "Ports_and_Firewalls_Requirements_{context}"
guides/common/modules/ref_ports-and-firewall-requirements.adoc:1:[id="Ports_and_Firewalls_Requirements_{context}"]

➜  foreman-documentation git:(master) cat guides/doc-Installing_Server_Disconnected/master.adoc | grep "^:context:"
:context: satellite
```

Reply:
> I assume this is due to disconnected being officially unsupported for Foreman/Katello.

Note that I think it could be supported, but the instructions aren't there. And repositories are completely different between the various build flavors.

Reply:
Anything for me to do here?

Reply:
No, this is ok!